app/controllers/broker_agencies/applicants_controller.rb

```diff
@@ -48,15 +48,15 @@ def update
       flash[:notice] = "Applicant accepted successfully."
     end
 
-    redirect_to broker_agencies_profile_path(@broker_agency_profile)
+    redirect_to edit_broker_agencies_profile_applicant(@broker_agency_profile, @broker_applicant)
```

Comment:
Missing "_path" at the end, should be edit_broker_agencies_profile_applicant_path